utils/check_repo.py

```diff
@@ -356,9 +363,13 @@ def check_all_models_are_tested():
     test_files = get_model_test_files()
     failures = []
     for module in modules:
-        test_file = f"test_{module.__name__.split('.')[-1]}.py"
-        if test_file not in test_files:
+        test_file = [file for file in test_files if f"test_{module.__name__.split('.')[-1]}.py" in file]
+        if not len(test_file):
```

Comment:
```suggestion
        if len(test_file) == 0:
```
Let's make the test explicit :-)